.gitignore

```diff
@@ -290,4 +290,6 @@ __pycache__/
 node_modules
 out
 *.vsix
-package-lock.json
\ No newline at end of file
+package-lock.json
+# temporarily ignore for incremental tslinting
+tslint.json
```

Comment:
Do you want to ignore this or not? Because the file is included in this PR...

Reply:
I wanted to ignore it, but it didn't work so I just took out the npm lint command in the travis.yml instead.  I think that worked.